main.go

```diff
@@ -49,8 +52,31 @@ func internalServerError(err error, c echo.Context) error {
 	return c.JSON(http.StatusInternalServerError, ErrorResponse{Message: err.Error()})
 }
 
-func identityJSONTransform(j JsonLike) JsonLike {
-	return j
+func identityJSONTransform(body []byte) (interface{}, error) {
+	var i interface{}
+	err := json.Unmarshal(body, &i)
+	if err != nil {
+		return nil, err
+	}
+
+	switch v := i.(type) {
+	case []interface{}:
+		// JSON array
+		jsonArray := make([]JsonLike, len(v))
+		for i, item := range v {
+			m, ok := item.(map[string]interface{})
+			if !ok {
```

Comment:
worth putting a comment/note here that this works for arrays of objects, but will break in the future if some endpoint returned an array of numbers/strings/etc - it's fine anyway, since we will eventually get rid of the go component